container/20231116.md

```diff
@@ -0,0 +1,58 @@
+# 2023-11-16
+## :family: Participants (github handles preferred)
+* @matofeder
+* @fdobrovolny
+* @janiskemper
+* @tonifinger
+* @batistein
+* @joshmue
+* @cah_hannesbaum
```

Comment:
his handle is: @cah-hbaum

Reply:
oh he probably has written that down himself? at least I haven't changed anything. Should I make a new PR for this fix?